client/my-sites/hosting/miscellaneous-card/index.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Button, Card, Dialog } from '@automattic/components';
+import FormLabel from 'components/forms/form-label';
+import CardHeading from 'components/card-heading';
+import MaterialIcon from 'components/material-icon';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { clearWordPressCache } from 'state/hosting/actions';
+import getRequest from 'state/selectors/get-request';
+import { isAtomicClearCacheEnabled } from 'state/selectors/is-atomic-clear-cache-enabled';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const MiscellaneousCard = ( {
+	disabled,
+	clearAtomicWordPressCache,
+	isClearCacheEnabled,
+	isClearingCache,
+	siteId,
+	translate,
+} ) => {
+	const [ showDialog, setShowDialog ] = useState( false );
+	const [ reason, setReason ] = useState( '' );
+
+	const clearCache = () => {
+		clearAtomicWordPressCache( siteId, reason );
+		setShowDialog( false );
+		setReason( '' );
+	};
+
+	const showConfirmationDialog = () => {
+		setShowDialog( true );
+	};
+
+	const closeConfirmationDialog = () => {
+		setShowDialog( false );
+	};
+
+	const onReasonChange = event => {
+		setReason( event.target.value );
+	};
+
+	const getClearCacheContent = () => {
+		const clearCacheText = translate( 'Clear Cache' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **9** times:
`translate( 'Clear cache on error.' )` *ES Score: 7*

Reply:
Not good for our use case.